tutorial/20_recipes/002_multi_objective.py

```diff
@@ -119,10 +119,23 @@ def objective(trial):
 
 
 ###################################################################################################
-# Check trials on pareto front visually
+# Check trials on pareto front visually.
 optuna.visualization.plot_pareto_front(study, target_names=["FLOPS", "accuracy"])
 
 
+###################################################################################################
+# Fetch the list of trials on the Pareto front with :attr:`~optuna.study.Study.best_trials`.
+#
+# For example, the following code picks the trial with the highest accuracy from the Pareto front.
+
```

Comment:
This section is rendered as follows:
![image](https://user-images.githubusercontent.com/3255979/161684471-766dab70-2f8b-4bd1-be85-efea7d38494f.png)

IMHO, the description and code example may be a bit inconsistent, so could you add more words for it?
The following is just an example.

```suggestion
# You can fetch the list of trials on the Pareto front with :attr:`~optuna.study.Study.best_trials`.
#
# For example, the following code checks the number of the trials on the Pareto front and picks the trial with the highest accuracy.

```

Reply:
Thanks, I'll fix this.

Reply:
I fixed it in [8297474](https://github.com/optuna/optuna/pull/3443/commits/8297474e7bdce7f7d376cb0031d0073124668e03).

Reply:
Thanks!